### PR TITLE
Disable 'import/prefer-default-exports'

### PR DESCRIPTION
**Pull request by matijs**
For the reasoning behind this decision, read:

https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,12 +0,0 @@
-*.DS_Store
```
Comment (xli12) [inline .npmignore]: Wondering why `.npmignore` is removed?

**Comment by xli12** [comment]
Is that mean in the future we prefer named exports rather than default exports?

File: .npmignore (hunk 1 above)
Comment (matijs) [reply .npmignore]: The only file that needs to be published is index.js so I put that in package.json

I have documented it in the [commit](https://github.com/Ultimaker/eslint-config/pull/9/commits/690ee8133ab335c8e62cad675d2c8c83445aebb6)

**Comment by matijs** [comment]
I think we should when it makes sense. There are tools/packages that expect default exports so we should not prohibit default exports.

**Comment by ianpaschal** [comment]
> https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

I find this thoroughly unconvincing. If it matters to you, Matijs, I'm fine to change it for work stuff but I think default is superior. Especially when using TypeScript his complaints are basically all non-issues.

**Comment by matijs** [comment]
In that case I'll merge and release a new version that will allow both default and named exports instead of preferring default ones.

**Pull request merged**